pkg/operation/common/utils.go

```diff
@@ -183,3 +184,20 @@ func CheckConfirmationDeletionTimestampValid(objectMeta metav1.ObjectMeta) bool
 	confirmationDeletionTimestamp := metav1.NewTime(timestamp)
 	return confirmationDeletionTimestamp.Equal(deletionTimestamp)
 }
+
+// ExtractShootName returns Shoot resource name extracted from provided <backupInfrastructureName>.
+//TODO confirm not needed and
+func ExtractShootName(backupInfrastructureName string) string {
+	tokens := strings.Split(backupInfrastructureName, "-")
+	return strings.Join(tokens[:len(tokens)-1], "-")
+}
+
+// GenerateBackupInfrastructureName returns BackupInfrastructure resource name created from provided <seedNamespace> and <shootUID>.
+func GenerateBackupInfrastructureName(seedNamespace string, shootUID types.UID) string {
+	return fmt.Sprintf("%s-%s", seedNamespace, utils.ComputeSHA1Hex([]byte(shootUID))[:5])
+}
+
+// GenerateBackupNamespaceName returns Backup namespace name created from provided <backupInfrastructureName>.
+func GenerateBackupNamespaceName(backupInfrastructureName string) string {
+	return fmt.Sprintf("%s-%s", BackupNamespacePrefix, backupInfrastructureName)
```

Comment:
Please use
`backup--<projectName>--<shootName>--<uid>` as namespace.
See #164 for details.